patches/server/0130-Cap-Entity-Collisions.patch

```diff
@@ -55,3 +55,25 @@ index 36aef3dbfe407ca6fa206bc3fc4390e5c2770aa3..e0fe53045da81fc9b2eac3b215d641cf
  
                  this.doPush(entity);
              }
+diff --git a/src/main/java/org/spigotmc/SpigotWorldConfig.java b/src/main/java/org/spigotmc/SpigotWorldConfig.java
+index af8c42ed04d9a79cd426bdaa8c2ee3feac1163e3..35f6619081e5a1053c9d655f18359927b2fc6d6b 100644
+--- a/src/main/java/org/spigotmc/SpigotWorldConfig.java
++++ b/src/main/java/org/spigotmc/SpigotWorldConfig.java
+@@ -57,9 +57,15 @@ public class SpigotWorldConfig
+     }
+ 
+     public int getInt(String path, int def)
++    // Paper start - get int without setting default
+     {
+-        this.config.addDefault( "world-settings.default." + path, def );
+-        return this.config.getInt( "world-settings." + this.worldName + "." + path, this.config.getInt( "world-settings.default." + path ) );
++        return this.getInt(path, def, true);
++    }
++    public int getInt(String path, int def, boolean setDef)
```

Comment:
not sure if this should be added to the general config patch? guess we can move it later if we need more of these

Reply:
yeah... I guess I'll move it now. There may be cases in the future where we want to do something similar.